cloud_governance/tag_non_cluster/tag_non_cluster_resources.py

```diff
@@ -274,11 +280,12 @@ def update_ami(self, images: list = None):
             search_tags.extend(self.__append_input_tags())
             if username:
                 user_tags = self.iam_client.get_user_tags(username=username)
-                tag_name = f'{username}-{image_id[-4:]}'
+                tag_name = f'{username}-{image_id[-5:]}'
```

Comment:
Please add constant for 5 with meaningful name

Reply:
ok

Reply:
LENGHT ??

Reply:
SHORT_ID